pallets/process-validation/src/restrictions.rs

```diff
@@ -26,6 +26,18 @@ pub enum Restriction<RoleKey, TokenMetadataKey, TokenMetadataValue, TokenMetadat
         index: u32,
         role_key: RoleKey
     },
+    OutputHasMetadataValue {
+        index: u32,
+        metadata_key: TokenMetadataKey
+    },
+    InputHasRole {
+        index: u32,
+        role_key: RoleKey
+    },
+    InputHasMetadataValue {
```

Comment:
should `HasMetadataValue` be `HasMetadataKey`?

Reply:
🤔 , maybe `HasMetadata`? Because it's whether it has that metadata entry

Reply:
yeah `HasMetadata` would match `HasRole`

Reply:
in that they have a similar name and are both checking there is a role/metadata entry with any value